### ISSUE TITLE
Own unique Package Name

### ISSUE THREAD

**Issue by hagst** [CocoCR300/flauncher#12]
I would like it if this version of FLauncher has his own unique Package Name.

**Comment by CocoCR300** [comment]
I don't want to change too much of those things about the app, if it diverges enough from the original I would, but it won't be a priority.

**Comment by bt4ibwem8** [comment]
Klauncher would be cool name for it 🙂. It could be first step for f-droid inclusion...

**Comment by hagst** [comment]
[Klauncher](https://play.google.com/store/apps/details?id=com.tranhuynh.klauncher) already exists.
Most names with one letter before Launcher already exist.
Perhaps "Thin Launcher", because it is a simple/small launcher but perhaps this looks to much as "thin client".

**Comment by celenityy** [comment]
Maybe something like "Coco Launcher"? 🤔

**Comment by bt4ibwem8** [comment]
@hagst 
You are right, i did not search on Play Store, only in F-droid. Anyway if the dev do not play to put Flauncher in Play store then theoretically he could use the name which is already taken in Play store...
But yes, it would be best to choose a unique name.
